packages/contracts-bedrock/snapshots/abi/OptimismSuperchainERC20.json

```diff
@@ -596,27 +557,27 @@
   },
   {
     "inputs": [],
-    "name": "InvalidCrossDomainSender",
+    "name": "InvalidInitialization",
     "type": "error"
   },
   {
     "inputs": [],
-    "name": "InvalidInitialization",
+    "name": "InvalidPermit",
     "type": "error"
   },
   {
     "inputs": [],
-    "name": "InvalidPermit",
+    "name": "NotInitializing",
     "type": "error"
   },
   {
     "inputs": [],
-    "name": "NotInitializing",
+    "name": "OnlyL2StandardBridge",
     "type": "error"
   },
   {
     "inputs": [],
-    "name": "OnlyBridge",
+    "name": "OnlySuperchainERC20Bridge",
```

Comment:
We want to move away from the superchain specific naming so that this can be an ethereum wide standard in theory

Reply:
This error is being used in the `SuperchainERC20` that is the OP implementation of `ICrosschainERC20`, the neutral namings are set that interface so that it can be used in other ethereum chains